src/vlt/src/start-gui.ts

```diff
@@ -224,11 +224,17 @@ const updateGraphData = (
 
 const updateDashboardData = async (
   tmp: string,
-  options: ConfigOptions,
+  conf: LoadedConfig,
 ) => {
+  const userDefinedProjectPaths =
+    // eslint-disable-next-line
```

Comment:
@lukekarrys had to end up adding this disable line comment otherwise eslint keeps removing the `?.` from the next line and tests will actually break without that 😕

Reply:
Looks like we're doing some manual casting with `as LoadedConfig` in the tests. According to the types `LoadedConfig['values']` always exists but when we cast we're suppressing that error in the tests. The autofix behavior is following the types, so it "fixes" it which actually breaks.

I added a band-aid in #335. We currently have a lot of linting rules turned off for tests. I wonder if those would help spot the unsafe behavior in these cases?